graylog2-web-interface/src/components/lookup-tables/DataAdapterForm.jsx

```diff
@@ -76,7 +116,31 @@ const DataAdapterForm = React.createClass({
       promise = LookupTableDataAdaptersActions.update(this.state.dataAdapter);
     }
 
-    promise.then(() => { this.props.saved(); });
+    promise.then(() => {
```

Comment:
Failed validation might include somthing like a missing file, which should not prevent the entity to be created, I think. Other structural things will be returned as an exception, which handling is missing yet, correct.